content/docs/1.2.3/deploy/install/_index.md

```diff
@@ -86,14 +86,42 @@ If MountPropagation is disabled, Base Image feature will be disabled.
 
 ### Root and Privileged Permission
 
-The longhorn manager uses `Bidirectional` mount propagation to access the host's disk (default `/var/lib/longhorn`) in the container.
-An use case is to access disk metadata and replicas directory on the host.
-
-Besides, Longhorn uses `nsenter` command to execute commands on the host. The `nsenter` command, for example, is to check the host's disk space usage
-```shell
-stat /var/lib/longhorn/ -fc '{"path":"%n","fsid":"%i","type":"%T","freeBlock":%f,"totalBlock":%b,"blockSize":%s}'
-```
-The `nsenter` command rely on privileged containers and `hostPID: true`.
+Longhorn components require root access with privileged permissions to achieve volume operations and management, because Longhorn relies on system resources on the host across different namespaces, for example, Longhorn uses `nsenter` to understand block devices' usage or encrypt/decrypt volumes on the host.
+
+Below are the list of the directories of all components are:
+
+- Longhorn Manager
+  - /dev: Longhorn is a block storage solution for Kubernetes. The block device is created under `/dev` path.
+  - /proc: To access host's mount namespace by `nsenter` across different namepsaces.
+  - /var/lib/longhorn: The path to use for storing data on a host.
+- Longhorn Engine
+  - /var/lib/longhorn/engine-binaries: The path where the Longhorn engine binaries located.
+- Longhorn Instance Manager (Engine)
+  - /dev: Longhorn is a block storage solution for Kubernetes. The block device is created under `/dev` path.
+  - /proc: To access host's mount namespace by `nsenter` across different namepsaces.
+  - /var/lib/longhorn/engine-binaries: The path where the Longhorn engine binaries located.
+- Longhorn Instance Manager (Replica)
+  - /: Access any data path on this node and access Longhorn engine binaries.
+- Longhorn Share Manager
+  - /dev: Longhorn is a block storage solution for Kubernetes. The block device is created under `/dev` path.
+  - /lib/modules: Kernel module requried by `cryptsetup`.
+  - /proc: To access host's mount namespace by `nsenter` across different namepsaces.
+  - /sys: To support volume encryption by `cryptsetup`.
+- Longhorn CSI Plugin
+  - /: For RWX shared volume.
```

Comment:
"/" should no longer be needed with the new nsenter shell script and the removed nfsmounter.
Since we are running with `hostPid: true` -> `/proc => /proc`

Reply:
Since within v1.2.3 Longhorn code, we still have mount point root mount point inside the CSI plugins.
Therefore, we should list the root mount point in the doc to let the user/customer knows the purpose from the security point of view.